OpenOversight/app/commands.py

```diff
@@ -209,20 +214,25 @@ def update_officer_field(fieldname):
             if row[fieldname] == "":
                 row[fieldname] = None
             old_value = getattr(officer, fieldname)
-            # If we're expecting a date type, attempt to parse row[fieldname] as a datetime
-            # This also normalizes all date formats, ensuring the following comparison works properly
+            # If we're expecting a date type, attempt to parse row[fieldname] as a
+            # datetime This also normalizes all date formats, ensuring the following
```

Comment:
The formatting of the comment seems a little weird now since the sentence wasn't ended with a period, only a newline.